src/gui/src/Director.cpp

```diff
@@ -65,7 +65,7 @@ using namespace H2Core;
 
 Director::Director ( QWidget* pParent )
 		: QDialog ( pParent )
-		, Object ( "Director" )
```

Comment:
The corresponding header file `Director.h` was not modified.

Hmm. Turns out it did not contain the `H2_OBJECT` macro. Could you add it?

Reply:
done